### PR TITLE
JENKINS-29516 : Display disk usage of `java.io.tmpdir` cc @reviewbybees

### PR DESCRIPTION
**Pull request by aheritier**

### CODE REVIEW

**Comment by aheritier** [comment]
please @reviewbybees
Here is the output it gives:

screenshot 2015-07-24 19 23 59

**Comment by oleg-nenashev** [comment]
I think it should be in another runnable with its own lifecycle. TMP dir usually contains millions of small files, hence it may cause a performance degradation.

**Comment by jtnord** [comment]
I'm not entirely sure what the point of showing this is when you have no option to actually do anything about it.
The free temp space is shown for the node on `${JENKINS_URL}/computer/(master)` and computed by [TemporarySpaceMonitor](http://javadoc.jenkins-ci.org/?hudson/node_monitors/TemporarySpaceMonitor.html)  so short of using cycles I'm not sure what this brings to the party?

File: src/main/java/com/cloudbees/simplediskusage/QuickDiskUsage.java [hunk 1]
```diff
@@ -152,6 +155,12 @@ public void run() {
 
                     Thread.sleep(1000); //To keep load average nice and low
                 }
+                File tmpDir = new File(System.getProperty("java.io.tmpdir"));
+                usage.put(new DiskItem(
+                                "java.io.tmpdir",
```
Comment (jtnord) [inline src/main/java/com/cloudbees/simplediskusage/QuickDiskUsage.java]: I don;t think a user will always know what this is - why not call it "System temporary files"

**Comment by jtnord** [comment]
@oleg-nenashev having this not done in a different lifecycle helps ensure that disks are not thrashed as the monitoring of jobs/tmp will not both run at the same time (tmp does not have to be on a separate disk)

**Comment by jtnord** [comment]
@alanharder :bug: on exposing "java.io.tmpdir" to the user, otherwise ok.

File: src/main/java/com/cloudbees/simplediskusage/QuickDiskUsage.java (hunk 1 above)
Comment (recena) [inline src/main/java/com/cloudbees/simplediskusage/QuickDiskUsage.java]: @aheritier I agree with @jtnord.

**Comment by aheritier** [comment]
superseded by PR #5

**Pull request closed without merging**